cfme/tests/infrastructure/test_timelines.py

```diff
@@ -24,12 +27,13 @@
 @pytest.fixture(scope='module')
 def new_vm(request, provider):
     vm = VM.factory(random_vm_name('timelines', max_length=16), provider)
-    logger.debug('Fixture new_vm set up! Name: %r', vm.name)
-    logger.info('Will create  %r on Provider: %r', vm.name, vm.provider.name)
-    vm.create_on_provider(find_in_cfme=False, timeout=500)
+    vm.create_on_provider(find_in_cfme=True)
+    vm.wait_to_appear()
```

Comment:
Required: can be rmoved as create_on_provider(find_in_cfme=True) already have wait_to_appear included